P6: RiscV32 syscall enum missing sched_rr_get_interval

Surfaced while running the cross-compile matrix for #495.

### Symptom
`pr-cross-compile (riscv32-linux-musl)` fails with:
```
lib/std/os/linux.zig:2596:22: error: enum 'os.linux.syscalls.RiscV32' has no member named 'sched_rr_get_interval'
    return syscall2(.sched_rr_get_interval, ...);
lib/std/os/linux/syscalls.zig:6167:21: note: enum declared here
pub const RiscV32 = enum(usize) {
```

### Root cause
`sched_rr_get_interval` is missing from the `RiscV32` syscall-number enum. The actual syscall number is 165 (from musl's `arch/riscv32/bits/syscall.h.in`: `__NR_sched_rr_get_interval_time64` is 165 on riscv32 since it's a `time64`-only arch; the legacy `time32` variant is omitted).

### Where to fix
- `lib/std/os/linux/syscalls.zig:6167` — add `sched_rr_get_interval = 165,` to the `RiscV32` enum
- Possibly also rename the user-facing call to `sched_rr_get_interval_time64` and have it call through the `time64` path

### Fix sketch
Cross-reference musl's `arch/riscv32/bits/syscall.h.in` and add any other missing syscall numbers in the same family while you're there.

### Priority
**P6** — one cross-compile target blocked.

### Context
- https://github.com/ctaggart/zig/issues/495#issuecomment-4415502404

--- a/lib/std/os/linux.zig
+++ b/lib/std/os/linux.zig
@@ -2593,7 +2593,11 @@ pub fn sched_getattr(pid: pid_t, attr: *sched_attr, size: usize, flags: usize) u
 }
 
 pub fn sched_rr_get_interval(pid: pid_t, tp: *timespec) usize {
-    return syscall2(.sched_rr_get_interval, @as(usize, @bitCast(@as(isize, pid))), @intFromPtr(tp));
+    return syscall2(
+        if (@hasField(SYS, "sched_rr_get_interval")) .sched_rr_get_interval else .sched_rr_get_interval_time64,
+        @as(usize, @bitCast(@as(isize, pid))),
+        @intFromPtr(tp),
+    );
 }
 
 pub fn sched_yield() usize {

